agents/che-core-api-agent/src/test/resources/run_launcher_bats_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
That would be cool but I can't see how to do `docker exec` using the docker-maven-plugin. It doesn't look so trivial anyway. Do you mind if we address that in a separate PR?